src/tracing_tool_output.ml

```diff
@@ -113,33 +113,46 @@ module Serve = struct
   ;;
 end
 
+type events_output_format =
+  | Sexp
+  | Binio
+
+type events_writer =
+  { format : events_output_format
+  ; writer : Writer.t
+  ; output_callstack : bool
+  ; callstack_compression_state : Callstack_compression.t
+  }
+
 type t =
   { serve : Serve.t
-  ; output : [ `Fuchsia of string | `Sexp of string ]
+  ; fxt_path : string
+  ; events_path : string option
+  ; output_callstack : bool
   }
 
-let store_path = function
-  | `Fuchsia store_path | `Sexp store_path -> store_path
-;;
-
 let param =
-  let%map_open.Command store_path =
+  let%map_open.Command fxt_path =
     let default = "trace.fxt" in
     flag
       "output"
       (optional_with_default default string)
       ~aliases:[ "o" ]
-      ~doc:[%string "FILE Where to output the trace. (default: '%{default}')"]
+      ~doc:
+        [%string
+          "FILE Fuchsia file to output the fuchsia trace to. (default: '%{default}')"]
+  and events_path =
+    flag
+      "events-output"
+      (optional string)
+      ~doc:[%string "FILE Sexp or binio file to output the trace events to."]
+  and output_callstack =
```

Comment:
When would a person want to enable this flag? When would they want to disable it? If the compiler feature you're working on always wants this enabled, just enable it by default and delete the flag.